frontend/src/types/index.ts

```diff
@@ -0,0 +1,15 @@
+export interface AppState {
+  auth: Auth
+}
+
+export interface Auth {
```

Comment:
This will probably change when we implement auth (#48), but this is fine for now.

Reply:
I'm a long-form name guy, so I would make the argument for calling this interface `AuthenticationState`.